Dockerfile.gitlab

```diff
@@ -96,3 +96,9 @@ RUN set -x \
 # Install Datadog CI
 RUN npm install -g npm@9.6.5
 RUN npm install -g @datadog/datadog-ci
+
+# Install Datadog Java tracer
+ENV DD_TRACER_FOLDER $PWD/dd-java-tracer
```

Comment:
To not make a confusion here with the `dd-java-tracer` module that we are using for APM in our code now I would change the name of this folder to `dd-hava-agent`. What do you think @0xnm

Reply:
I'm fine with any, it won't be inside a project folder anyway (it will be inside the entry directory of the docker container).